diff --git a/cmd/zoekt-archive-index/BUILD.bazel b/cmd/zoekt-archive-index/BUILD.bazel
index f5553ff..fb945b2 100644
--- a/cmd/zoekt-archive-index/BUILD.bazel
+++ b/cmd/zoekt-archive-index/BUILD.bazel
@@ -7,7 +7,7 @@ go_library(
         "main.go",
     ],
     importpath = "github.com/sourcegraph/zoekt/cmd/zoekt-archive-index",
-    visibility = ["//visibility:private"],
+    visibility = ["//visibility:public"],
     deps = [
         "//cmd",
         "//internal/archive",
