diff --git a/cmd/zoekt-sourcegraph-indexserver/BUILD.bazel b/cmd/zoekt-sourcegraph-indexserver/BUILD.bazel
index 0e817fe..c447008 100644
--- a/cmd/zoekt-sourcegraph-indexserver/BUILD.bazel
+++ b/cmd/zoekt-sourcegraph-indexserver/BUILD.bazel
@@ -19,7 +19,7 @@ go_library(
     ],
     embedsrcs = ["default_grpc_service_configuration.json"],
     importpath = "github.com/sourcegraph/zoekt/cmd/zoekt-sourcegraph-indexserver",
-    visibility = ["//visibility:private"],
+    visibility = ["//visibility:public"],
     deps = [
         "//:zoekt",
         "//build",
