diff --git a/cmd/zoekt-git-index/BUILD.bazel b/cmd/zoekt-git-index/BUILD.bazel
index d924f9e..120a792 100644
--- a/cmd/zoekt-git-index/BUILD.bazel
+++ b/cmd/zoekt-git-index/BUILD.bazel
@@ -4,7 +4,7 @@ go_library(
     name = "zoekt-git-index_lib",
     srcs = ["main.go"],
     importpath = "github.com/sourcegraph/zoekt/cmd/zoekt-git-index",
-    visibility = ["//visibility:private"],
+    visibility = ["//visibility:public"],
     deps = [
         "//cmd",
         "//ctags",
