diff --git a/cmd/zoekt-indexserver/BUILD.bazel b/cmd/zoekt-indexserver/BUILD.bazel
index 761009d..166b73f 100644
--- a/cmd/zoekt-indexserver/BUILD.bazel
+++ b/cmd/zoekt-indexserver/BUILD.bazel
@@ -7,7 +7,7 @@ go_library(
         "main.go",
     ],
     importpath = "github.com/sourcegraph/zoekt/cmd/zoekt-indexserver",
-    visibility = ["//visibility:private"],
+    visibility = ["//visibility:public"],
     deps = [
         "//:zoekt",
         "//gitindex",
